### ISSUE TITLE
Image sizes

### ISSUE THREAD

**Issue by wpalchemist** [wpalchemist/pressable-trial#8]
@threegreenbirds The theme that I am using for your site has a slot for featured images that is 1180px wide by 435px tall.  The images you provided for me are 1000px wide, so they are too small for this space and look really strange (see screenshot below).

There are a few different ways we can solve this:

1. If you have larger versions of those images, send them to me and I can use them instead.
1. I can find some different images that are bigger.
1. If you don't have larger versions, but want to keep using the images you provided, I can either remove the large featured image at the top of each page, or adjust the theme so that the smaller images don't look awkward.

Which solution do you prefer?

![image](https://user-images.githubusercontent.com/16658165/38692016-6252bd24-3e37-11e8-89a6-4cb9401227fb.png)

**Comment by threegreenbirds** [comment]
Hmm, thanks. For now can you just make the images we provided bigger and we can find some better ones post-launch? If that looks bad, glad to have you find some similar, larger placeholder images for now.

**Comment by wpalchemist** [comment]
@threegreenbirds For now, I have just enlarged the images you provided.  Some of them look better than others enlarged.  I did replace the image of the green bird - I like the idea of having an image of a green bird on your "about" page, but the image you provided was proportioned in such a way that the bird's head got cut off when used as the featured image, so I replaced it with a different one.  It's easy to swap out images, so if you aren't happy with the one I found, we can find another one.
